packages/composite-checkout/README.md

```diff
@@ -315,6 +342,10 @@ A React Hook that will return true if the current step is complete as defined by
 
 A React Hook that will return a two element array where the first element is the current array of line items (matching the `items` prop on `Checkout`), and the second element is the current total (matching the `total` prop). Only works within [CheckoutProvider](#CheckoutProvider).
 
+### useLineItemsOfType
+
+A React Hook that will return an array of line items matching a specific 'type' (i.e. 'tax'). Only works within [CheckoutProvider](#CheckoutProvider).
```

Comment:
We should probably explain that it takes a string argument and what the `type` means. I'd say something like:

```md
A React Hook taking one string argument that will return an array of [line items](#line-items) from the cart (derived from the same data returned by [useLineItems](#useLineItems)) whose `type` property matches that string. Only works within [CheckoutProvider](#CheckoutProvider).
```